README.md

```diff
@@ -87,68 +112,72 @@ WSGIPythonPath usr/local/devcycle
 </VirtualHost>
 ```
 
-* Change the application settings:
+* Create and Change the application settings:
 
 ```
-nano /usr/local/devcycle/dataCollection/settings.py
+sudo cp /usr/local/devcycle/dataCollection/settings.py.template /usr/local/devcycle/dataCollection/settings.py
+sudo vim /usr/local/devcycle/dataCollection/settings.py
 ```
 
 Set DEBUG to False
-Under DATABASES, modify USER and PASSWORD fields to reflect user created in SETUP (above).
+Under DATABASES, modify USER and PASSWORD fields to reflect the database user you created in "Install The Database" (above).
 HOST should be localhost.
-STATIC_ROOT to point to '/var/www/static' unless you modified where to collect these in 
+STATIC_ROOT to point to '/var/www/static' unless you modified where to collect these in
 step 6.
 SECRET_KEY is a string of at least 32 random characters
 KEY to a random string of hex characters a multiple of 16 long
 SECRET to a random string of numeric characters a multiple of 16
 STATIC_URL = '/static/'
 STATICFILES_DIRS = (
-  '‘/usr/local/devcycle-server/tour_config/static/’
+  '‘/usr/local/devcycle/tour_config/static/’
   )
-  
-Restart the apache server to put all changes into effect. 
+
+Restart the apache server to put all changes into effect.
 
 ```
-/etc/init.d/apache2 reload
+sudo /etc/init.d/apache2 reload
 ```
 
-###Migrate the Database Schema using South
+
+###Setup The Database
 [South](http://south.aeracode.org/) is a schema and data migration tool for Django. It is used for easily
 migrating the database schema from database to database if needed. It is also used in the case of making updates
 to models then wanting those changes reflected in the database schema. South is already installed if you ran the `bash setup.sh` command. Recommend looking at the docs for more information [docs](http://south.readthedocs.org/en/latest/index.html)
 
-At this point the database schema for the Server does not exist yet. We will use South to add it. South will look at the current models to set-up the schema that the Server requires. 
+At this point the database schema for the Server does not exist yet. We will use South to add it. South will look at the current models to set-up the schema that the Server requires.
 
-*Note all commands below need to be ran within the root directory of the Django Project*
+*Note all commands below need to be ran within the root directory of the Django Project (/usr/local/devcycle)*
 
 This command will create the migrations:
 
-`./manage.py makemigrations`
+`sudo python manage.py syncdb --all`
+
+ - _**Note:** this may prompt you to create a django auth user. Follow that process as well if it does._
 
 These commands will then apply the newly created migrations to the database:
 
-`./manage.py migrate rider 0001
+`sudo python manage.py migrate rider 0001 --fake`
 
-./manage.py migrate location_update 0001
```

Comment:
These aren't necessary for initial setup and you also lost the create migrations step. It might be better to add these as an addendum to the section.